packages/lesswrong/server/notificationCallbacks.tsx

```diff
@@ -508,11 +520,11 @@ getCollectionHooks("Messages").newAsync.add(async function messageNewNotificatio
   const recipientIds = conversation.participantIds.filter((id) => (id !== message.userId));
 
   // Create notification
-  await createNotifications(recipientIds, 'newMessage', 'message', message._id);
+  await createNotifications({userIds: recipientIds, notificationType: 'newMessage', documentType: 'message', documentId: message._id, noEmail: message.noEmail});
 });
 
 export async function bellNotifyEmailVerificationRequired (user: DbUser) {
-  await createNotifications([user._id], 'emailVerificationRequired', null, null);
+  await createNotifications({userIds: [user._id], notificationType: 'emailVerificationRequired', documentType: null, documentId: null});
```

Comment:
I think you don't have to specify "documentType: null" here, the whole point of converting to object-function-variables is that we don't need that anymore